sqladmin/application.py

```diff
@@ -250,7 +250,7 @@ def __init__(
         base_url: str = "/admin",
         title: str = "Admin",
         logo_url: str = None,
-        middlewares: Sequence[Middleware] = None,
+        middlewares: Optional[Sequence[Middleware]] = None,
```

Comment:
The `add_middleware` method will create a Middleware instance from it, it should only pass the middleware class type.

```suggestion
        middlewares: Optional[Sequence[type]] = None,
```